content/api/inline-seeds.apib

```diff
@@ -393,6 +393,33 @@ A user can turn off Automatic Inline Seeding for all subaccounts (including the
         }
         ```
 
+        + Request (application/json)
+
++ Request (application/json)
+
+    + Headers
+
+            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+            X-MSYS-SUBACCOUNT: 1
```

Comment:
I think that we should omit the subaccount header in this example because we're listing all options

Reply:
Good call. I've made the change